### ISSUE TITLE
Better error message when SSL certificate from daemon changes making wallet non-responsive

### ISSUE THREAD

**Issue by Doy-lee** [loki-project/loki#643]
When the daemon the wallet is connected to has its SSL certificate changed, the wallet errors out with

```
Error: wallet failed to connect to daemon: http://localhost:38154. Daemon either is not started or wrong port was passed. Please make sure daemon is running or change the daemon address using the 'set_daemon' command.
```

Which does not really give the correct information on what's going on. In particular if you're connected to a locally run daemon and you can physically see the daemon running after restarting it- it's not clear that the wallet refuses to use it because the SSL connection it has going on has changed.

**Comment by jagerman** [comment]
This wallet freezing (which happened every time I restarted lokid) seems to be gone now with #651 merged.

**Comment by Doy-lee** [comment]
Confirmed fixed in #651
